src/style.css

```diff
@@ -1 +1,36 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
```

Comment:
The `align-content` property is used for multi-line flex containers. Since the `.stars` container is likely a single line, this property may not have any effect. Consider removing it if it's not needed.